adr/recipes/config_durations.py

```diff
@@ -7,22 +7,54 @@
 """
 from __future__ import print_function, absolute_import
 
-from ..recipe import RecipeParser
-from ..query import run_query
+from ..recipe import get_argument_groups
+from ..recipe import execute_query
 
+QUERY_ARGUMENTS = [
```

Comment:
I'd prefer if there were a single global config variable here. Maybe something like:

    CONFIG = {
        'query_context': [
            'branch',
            'date',
            { ... },
        'args': [ ... ],
    }

Reply:
I removed the `QUERY_ARGUMENTS`. The query context definition is extracted automatically from the `*.query` file.